src/pages/SearchResults/components/CategoryTabBar.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from '@emotion/styled';
+import { useState } from 'react';
+
+interface TapWrapperProps {
+  isActive: boolean;
+}
+
+const CategoryTabBar = () => {
+  const [onActive, setOnActive] = useState(0);
+
+  return (
+    <Wrapper>
+      <TapWrapper isActive={onActive === 0} onClick={() => setOnActive(0)}>
```

Comment:
```suggestion
{categoryList.map((tab, _) => (
  <TapWrapper isActive={onActive === 0} onClick={() => setOnActive(0)}>
))
}
```

리스트로 빼고 map을 이용하여 동적 렌더링 해주세요